pkgs/development/libraries/ipu6-camera-hal/default.nix

```diff
@@ -38,6 +38,7 @@ stdenv.mkDerivation {
   ];
 
   NIX_CFLAGS_COMPILE = [
+    "-Wno-error"
```

Comment:
I first tried to only disable the actual warnings that showed up. One of those was `unused-but-set-variable` which can be disabled. But the other warning was related to redefinition of `_FORTIFY_SOURCE` and I found no way to disable it short of disabling all warnings as errors.